setup.py

```diff
@@ -58,10 +58,10 @@ def get_version():
     author_email="mbocek@redhat.com",
     url="www.redhat.com",
     license="GNU General Public License v3 or later (GPLv3+)",
-    packages=["convert2rhel"],
+    packages=["convert2rhel", "convert2rhel/pkgmanager", "convert2rhel/pkgmanager/handlers"],
```

Comment:
Why do we need to include this, aren't they covered by `convert2rhel` package?

Reply:
When I included those folders and generated the rpm, they were not included (both locally and packit generation)… That's why I manually included them, I guess the recursion doesn't go that further.